integrations/venice-duckdb/src/main/java/com/linkedin/venice/sql/AvroToSQL.java

```diff
@@ -0,0 +1,146 @@
+package com.linkedin.venice.sql;
+
+import java.sql.JDBCType;
+import java.util.Collections;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import org.apache.avro.Schema;
+
+
+/**
+ * Utility intended to convert Avro -> SQL, including DDL and DML statements.
+ *
+ * Initially, this implementation may have a DuckDB slant, though in the long-run it should ideally be vendor-neutral.
+ */
+public class AvroToSQL {
```

Comment:
Quick question:
Are you trying to build a table to cover all the top-level columns or the nested columns as well? 

Also what is the target use case in your mind?

Reply:
For now it only supports top-level primitive types, but it should be possible to evolve this incrementally to support nested records and collections as well...

The near-term goal is to use DVRT to load Venice data into DuckDB.